bitnami/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 0.0.2
+version: 1.0.0
```

Comment:
I would not update the major version in this case as the airflow chart has not reached the version 1.0.0 and is not considered stable yet.

Use 0.0.3 instead

Reply:
Done